src/orion/core/worker/trial.py

```diff
@@ -205,6 +205,7 @@ class Param(Value):
 
     def __init__(self, **kwargs):
         """See attributes of `Trial` for meaning and possible arguments for `kwargs`."""
+        kwargs.pop("owner_id", None)
```

Comment:
Perhaps it should be the database's responsibility to remove the `owner_id` if we are not supposed to be aware of it externally anyway.